components/director/internal/model/capability.go

```diff
@@ -0,0 +1,154 @@
+package model
+
+import (
+	"encoding/json"
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
+	"github.com/kyma-incubator/compass/components/director/pkg/accessstrategy"
+	"github.com/kyma-incubator/compass/components/director/pkg/resource"
+	"github.com/kyma-incubator/compass/components/director/pkg/str"
+	"regexp"
+	"strconv"
+)
+
+// Capability missing godoc
+type Capability struct {
+	ApplicationID                *string
+	ApplicationTemplateVersionID *string
+	PackageID                    *string
+	Name                         string
+	Description                  *string
+	OrdID                        *string
+	Type                         string
+	CustomType                   *string
+	LocalTenantID                *string
+	ShortDescription             *string
+	SystemInstanceAware          *bool
+	Tags                         json.RawMessage
+	Links                        json.RawMessage
+	ReleaseStatus                *string
+	Labels                       json.RawMessage
+	Visibility                   *string
+	Version                      *Version
+	ResourceHash                 *string
+	DocumentationLabels          json.RawMessage
+	CorrelationIDs               json.RawMessage
+	*BaseEntity
+}
+
+// GetType missing godoc
+func (*Capability) GetType() resource.Type {
+	return resource.Capability
+}
+
+// CapabilityInput missing godoc
+type CapabilityInput struct {
```

Comment:
By ord spec i I think there are missing `lastUpdate` field. This field needs to be added to APIs,Events and Bundles.
Also we need to add the necessary validations for this fields. For example:
lastUpdate - must be checked if it is in the RFC3339 time format. 
Also for `lastUpdated` field we need to add logic when the the `lastUpdate` value has not changed since last fetch, the resource definitions do not need to be fetched.

Reply:
Added `lastUpdate` field to API, Event and Capability with validation for checking if date is valid.